modules/secrets-store-azure.adoc

```diff
@@ -0,0 +1,278 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets-store.adoc
+
+:_content-type: PROCEDURE
+[id="secrets-store-azure_{context}"]
+= Mounting secrets from Azure Key Vault
+
+You can use the {secrets-store-operator} to mount secrets from Azure Key Vault to a CSI volume in {product-title}. To mount secrets from Azure Key Vault, your cluster must be installed on Microsoft Azure.
+
+.Prerequisites
+
+* Your cluster is installed on Azure.
+* You have installed the {secrets-store-operator}. See _Installing the {secrets-store-driver}_ for instructions.
+* You have configured Azure Key Vault to store the required secrets.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Install the Azure Key Vault provider:
+
+.. Create a YAML file with the following configuration for the provider resources:
++
+.Example `azure-provider.yaml` file
```

Comment:
Hi @bergerhoffer , for azure-provider.yaml Can you please take from this?
https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/master/deployment/provider-azure-installer.yaml

Reply:
@ropatil010 As I mentioned in my other comment, it was originally based on that file, and then modified in the ways I noted (see comments in line 139) to 1). Add what it needs to work with OCP, and 2). align with the AWS procedure